src/Blueprint/Blueprint/Cmdlets/NewAzureRMBlueprintAssignment.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Microsoft.Azure.Commands.Blueprint.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Blueprint.Models;
+using System;
+using System.Collections;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Language;
+using Microsoft.Azure.PowerShell.Cmdlets.Blueprint.Properties;
+using ParameterSetNames = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterSetNames;
+using ParameterHelpMessages = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterHelpMessages;
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Commands.Blueprint.Common;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintAssignment", SupportsShouldProcess = true)]
```

Comment:
>ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintAssignment" [](start = 29, length = 77)

I'm wondering if we should had a Nouns class to avoid weird inconsistencies in spelling or capitalization (didn't find any, though).

Reply:
Sounds good. I'll add that in the next phase of the cmdlets.

---
In reply to: [257119476](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257119476) [](ancestors = 257119476)